atlasdb-config/src/main/java/com/palantir/atlasdb/config/TimeLockClientConfig.java

```diff
@@ -27,4 +30,14 @@
     public abstract String client();
 
     public abstract ServerListConfig serversList();
+
+    public ServerListConfig toNamespacedServerList() {
```

Comment:
nit: could be @Derived but since it's not requested often at all this won't make much of a difference.